src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedIntrinsicOperatorSymbol.cs

```diff
@@ -18,9 +18,10 @@ internal sealed class SynthesizedIntrinsicOperatorSymbol : MethodSymbol
         private readonly string _name;
         private readonly ImmutableArray<ParameterSymbol> _parameters;
         private readonly TypeSymbol _returnType;
-        private readonly bool _isCheckedBuiltin;
 
-        public SynthesizedIntrinsicOperatorSymbol(TypeSymbol leftType, string name, TypeSymbol rightType, TypeSymbol returnType, bool isCheckedBuiltin)
+        public override bool IsCheckedBuiltin { get; }
```

Comment:
>IsCheckedBuiltin [](http://example.com/codeflow?start=29&length=16)

Let's change it to a calculated property, the name is always available.

Reply:
Sounds good